XPNet.CLR/Instance/XPlaneInstance.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace XPNet
+{
+	public interface IXPlaneInstance
+	{
+		unsafe IXPInstance Create(IXPSceneryObject inSceneryObject, IEnumerable<string> inDataRefs);
+	}
+
+	internal class XPlaneInstance : IXPlaneInstance
+	{
+		public IXPInstance Create(IXPSceneryObject inSceneryObject, IEnumerable<string> inDataRefs)
+		{
+			return inSceneryObject.CreateInstance(inDataRefs);
```

Comment:
@jaurenq what do you think about this solution? Now we we can "redirect" users, who want to create the instance via the instance API, so they can use it in a similar way as they would have done it in the SDK, they can also instantiate directly from the XPSceneryObject via CreateInstance, and we do not need to expose the object reference, keeping it private in the XPSceneryObject.